extensions/amp-user-notification/0.1/amp-user-notification.js

```diff
@@ -126,6 +125,11 @@ export class AmpUserNotification extends AMP.BaseElement {
       assertHttpsUrl(this.dismissHref_, this.element);
     }
 
+    const alwaysShowAttr = this.element.getAttribute('always-show');
+    /** @private @const {boolean} */
+    this.alwaysShow_ = (this.element.hasAttribute('always-show') &&
+        alwaysShowAttr != '0' && alwaysShowAttr != 'false');
```

Comment:
also for contenteditable:  "The empty string and the true keyword map to the true state. The false keyword maps to the false state. In addition, there is a third state, the inherit state, which is the missing value default (and the invalid value default)"